modules/testcontainers/src/main/kotlin/jp/co/soramitsu/iroha2/testcontainers/IrohaContainer.kt

```diff
@@ -97,8 +102,9 @@ open class IrohaContainer : GenericContainer<IrohaContainer> {
 
     companion object {
         const val NETWORK_ALIAS = "iroha"
-        const val DEFAULT_IMAGE_TAG = "stable"
-        const val IMAGE_NAME = "hyperledger/iroha2"
+        val DEFAULT_IMAGE_TAG = System.getenv("IROHA_IMAGE_TAG") ?: "stable"
```

Comment:
to minimize risk of collision I'd suggest to change `IROHA` prefix to `IROHA_JAVA` or smth